api/envoy/config/filter/udp/udp_proxy/v2alpha/udp_proxy.proto

```diff
@@ -10,7 +10,9 @@ import "google/protobuf/duration.proto";
 
```

Comment:
When doing an initial build of the UDP filters I get a message saying that the `import "google/protobuf/duration.proto";` above this line is not used.

Reply:
Hmm, it's used below for `idle_timeout`?

Reply:
Ah yes. The `idle_timeout` isn't in master yet and hence why the error is thrown. Ignore.